### PR TITLE
fixes to Clustering 3K PBMCs with Scanpy part II

### PR DESCRIPTION
**Pull request by hrhotz**
small improvements in the second half of the text in order to make it clearer which tool to use (same problem as #1760)

### CODE REVIEW

File: topics/transcriptomics/tutorials/scrna-scanpy-pbmc3k/tutorial.md [hunk 1]
```diff
@@ -871,7 +871,7 @@ Here we perform the PCA on the log-normalized expression values and compute the
 
 > ### {% icon hands_on %} Hands-on: Perform the PCA
 >
-> 1. **Cluster, infer trajectories and embed** {% icon tool %} with the following parameters:
+> 1. **Cluster,** infer trajectories and embed with scanpy {% icon tool %} with the following parameters:
```
Comment (mtekman) [inline topics/transcriptomics/tutorials/scrna-scanpy-pbmc3k/tutorial.md]: The tool name was updated to have the larger title.

See:
https://github.com/galaxyproject/tools-iuc/blob/9d7adf6b6dd22f25379336d2ab94bba513d9ad89/tools/scanpy/cluster_reduce_dimension.xml

File: topics/transcriptomics/tutorials/scrna-scanpy-pbmc3k/tutorial.md [hunk 2]
```diff
@@ -1152,7 +1152,7 @@ Here, we will reduce the neighborhood to 2 UMAP components and then we will chec
 
 > ### {% icon hands_on %} Hands-on: Embed and plot the neighborhood graph
 >
-> 1. **Cluster, infer trajectories and embed** {% icon tool %} with the following parameters:
+> 1. **Cluster,** infer trajectories and embed with scanpy {% icon tool %} with the following parameters:
```
Comment (mtekman) [inline topics/transcriptomics/tutorials/scrna-scanpy-pbmc3k/tutorial.md]: Also here

File: topics/transcriptomics/tutorials/scrna-scanpy-pbmc3k/tutorial.md [hunk 3]
```diff
@@ -1210,7 +1210,7 @@ Currently, the Louvain graph-clustering method (community detection based on opt
 
 > ### {% icon hands_on %} Hands-on: Cluster the neighborhood graph
 >
-> 1. **Cluster, infer trajectories and embed** {% icon tool %} with the following parameters:
+> 1. **Cluster,** infer trajectories and embed with scanpy {% icon tool %} with the following parameters:
```
Comment (mtekman) [inline topics/transcriptomics/tutorials/scrna-scanpy-pbmc3k/tutorial.md]: And finally here :-)

File: topics/transcriptomics/tutorials/scrna-scanpy-pbmc3k/tutorial.md (hunk 1 above)
Comment (hrhotz) [reply topics/transcriptomics/tutorials/scrna-scanpy-pbmc3k/tutorial.md]: ...but on https://usegalaxy.eu/ , there is still only "cluster" bold, isn't-it?

Comment (hrhotz) [reply topics/transcriptomics/tutorials/scrna-scanpy-pbmc3k/tutorial.md]: ...I see, now it has changed. I will modify my pull request accordingly

Comment (mtekman) [reply topics/transcriptomics/tutorials/scrna-scanpy-pbmc3k/tutorial.md]: Yeah I think the tools weren't refreshed earlier -- thanks for this

**Pull request merged**